index.Rmd

```diff
@@ -466,13 +463,31 @@ library(sf)
 source('R/function1.R')
 source('R/function2.R')
 # Load datasets 
-load('data/input_model.rda')
+wageningen_ndvi <- rast('data/wag_NDVI.tif')
+# Then the actual commands
+```
+
+#### and in Python
+
+```{python, eval=FALSE}
+# Team Teamname (John Doe and Jane Smith)
+# January 2020
+# Import packages
+import geopandas as gpd
+import matplotlib.pyplot at plt
+
+# Import functions
+import Python.functions as funcs
+
+# Load datasets 
+postboxes = gpd.read_file('data/postbox_locations.shp')
 # Then the actual commands
 ```
 
-### Working directory, relative and absolute file paths
+## Working directory, relative and absolute file paths
+*At the end of the following section you should be able to explain the difference between __root directory, working directory, absolute path, relative path__* 
 
-As you can see in the example file above, we source a file from our project using **relative** file paths. The reason why they are called *relative* is because they are *relative* to the **working directory**, which is the directory *from which the script is called*. By convention, the working directory is the same as the location of the script itself. This means that you can simply assume that whoever runs your script, will run it from the directory that your script is located.
+In the R and Python examples above we load the datasets by indicating the file location from the data directory `"data/postbox_locations.shp"`. This is an example of using **relative** file paths. We don't include the location of the project directory itself (`"C:/Geoscripting/myproject/data/postbox_locations.sph"`). The reason why they are called *relative* is because they are *relative* to the **working directory**, which is the directory *from which the script is called* (`"C:/Geoscripting/myproject"`). By convention, the working directory is the same as the location of the script itself. This means that you can simply assume that whoever runs your script, will run it from the directory that your script is located.
```

Comment:
The postbox locations are now only in the Python example. You could make them the same by making the R example use `st_read` as well (though a `.shp` is perhaps not the best example, we could do `.gpkg`).

Also, right now it's still not entirely clearly written, i.e. we should also mention why including absolute paths is a bad idea. If it's easier to do, you can rewrite the paragraph to make it flow better.